src/Utils/resizer.ts

```diff
@@ -8,8 +9,32 @@ const services = configureImageServices({
   gemini: {
     endpoint: GEMINI_CLOUDFRONT_URL,
   },
+  lambda: {
+    endpoint: "https://d1j88w5k23s1nr.cloudfront.net",
+    sources: [
+      {
+        source: "https://d32dm0rphc51dk.cloudfront.net",
+        bucket: "artsy-media-assets",
+      },
+      {
+        source: "https://files.artsy.net",
+        bucket: "artsy-vanity-files-production",
```

Comment:
@mzikherman and I reconfigured the Lambda to support this as an additional source.